contracts/UFragmentsPolicy.sol

```diff
@@ -210,6 +210,20 @@ contract UFragmentsPolicy is Ownable {
         rebaseWindowLengthSec = rebaseWindowLengthSec_;
     }
 
+    /**
+     * @notice A multi-chain AMPL interface method. The Ampleforth monetary policy contract
+     *         on the base-chain and XCAmpleController contracts on the satellite-chains
+     *         implement this method. It returns what the current contract believes to be
```

Comment:
"It returns what the...." 
->
"It atomically returns two values: what the current..."

Reply:
👍